### PR TITLE
add image pull secrets to the EventLogger api

### PR DESCRIPTION
**Pull request by xosk31**

### CODE REVIEW

File: helm/crds/eventlogger.bakito.ch_eventloggers.yaml [hunk 1]
```diff
@@ -51,6 +51,29 @@ spec:
...
+                  ImagePullSecrets is an optional list of references to secrets in the same namespace to use for pulling any of the images used by this EventLoggerSpec.
+                  If specified, these secrets will be passed to individual puller implementations for them to use.
+                items:
+                  description: |-
+                    LocalObjectReference contains enough information to let you locate the
+                    referenced object inside the same namespace.
+                  properties:
+                    name:
+                      default: ""
+                      description: |-
```
Comment (bakito) [inline helm/crds/eventlogger.bakito.ch_eventloggers.yaml:65]: Is there a possibility to have proper descriptions generated here?
The TODO's do not look so pretty.

**Comment by bakito** [comment]
@xosk31 Thank you for ypur contribution

File: helm/crds/eventlogger.bakito.ch_eventloggers.yaml (hunk 1 above)
Comment (xosk31) [reply helm/crds/eventlogger.bakito.ch_eventloggers.yaml:65]: It is automatically generated by the controller-gen

Comment (bakito) [reply helm/crds/eventlogger.bakito.ch_eventloggers.yaml:65]: I'll check if I can fix it after generation

**Pull request merged**